doc/matlab_visualization.md

```diff
@@ -0,0 +1,43 @@
+# How to use Matlab Visualization
+The Matlab Visualization uses the Matlab capabilities for displaying meshes and figures and the floating-base kinematics of iDyntree to get the new poses of the links.
+It uses the [iDyntreewrappers](https://github.com/robotology/idyntree/tree/master/bindings/matlab/%2BiDynTreeWrappers) to handle the model and obtaining the required transforms.
```

Comment:
Avoid absolute links. You can just use relative links and they work fine as well.